package.json

```diff
@@ -40,10 +40,11 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "7.23.9",
-    "@babel/plugin-transform-regenerator": "7.23.3",
-    "@babel/preset-env": "7.23.9",
-    "@babel/preset-react": "7.23.3",
+    "@babel/core": "7.28.4",
+    "@babel/plugin-syntax-dynamic-import": "7.8.3",
```

Comment:
```suggestion
```

I think that this is not needed anymore, according to this: https://github.com/geosolutions-it/MapStore2/issues/11603 
And due to the [PR](https://github.com/geosolutions-it/MapStore2/pull/11604/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R44) ` "@babel/plugin-transform-regenerator": "7.23.3",` looks to be included, but looking at [doc](https://babeljs.io/docs/babel-plugin-transform-regenerator) it looks to be already present in `/preset-env`

I think is an error of merge.